packages/wdio-sync/src/wrapCommand.js

```diff
@@ -49,6 +49,8 @@ export default function wrapCommand (commandName, fn) {
             inFiber(this)
             return futureResult
         } catch (e) {
+            inFiber(this)
```

Comment:
Do you know why should it be moved for multiremote custom commands?

moving `inFiber` before returning a result affects performance and might cause a regression of https://github.com/webdriverio/webdriverio/issues/4133

Reply:
@mgrybyk I just ran the example from the issue and had similar results as to what is expected from the fast example. So I think this change didn't affect the performance of it.

Reply:
This change is responsible for allowing user to run commands from single instances in multiremote synchronously. Before this would return only a promise.

Reply:
Got it, I want to verify it locally today

Reply:
Ok, I've verified it locally and have to say that it affects performance.

Before the change I have 15 beforeCommand calls and after the change 44.

My test:
```
describe("main suite", () => {
    beforeEach(() => {
        browser.url("http://guinea-pig.webdriver.io/")
    })

    it("remove before wait for not exist", () => {
        const el = $$('body')[0].$('aside').$('span')

        browser.execute(function (el) { el.remove() }, el)
        el.waitForNotDisplayed(2000)
    })
})
```

Reply:
I'm not able to reproduce defect locally after reverting back position of `inFiber(this)`.

However smoke tests failing for me in such case.

Here is my test that is passing in my working project:
```
   it('mr', () => {
        // @ts-ignore
        assert.equal(browser.browserB.getTitle(), 'WebdriverJS Testpage')
        // @ts-ignore
        assert.equal(browser.browserA.getTitle(), 'WebdriverJS Testpage')
        // @ts-ignore
        // const browserA = browser
        browserA.addCommand('testCommand', function () {
            // @ts-ignore
            browserA.url('http://guinea-pig.webdriver.io/');
            // @ts-ignore
            let el = browserA.$('aside');
            // console.log(el);
            console.log(el.getHTML());
        });
        // @ts-ignore
        browserA.testCommand();

        // @ts-ignore
        global.browserA.addCommand('foobar', () => {
            // @ts-ignore
            const title = global.browserA.getTitle()
            return `Title: ${title}`
        })
        // @ts-ignore
        assert.strictEqual(global.browserA.foobar(), 'Title: WebdriverJS Testpage')


        // @ts-ignore
        browserA.overwriteCommand('getTitle', function (origCommand: any) {
            return `Title: ${origCommand()}`
        })
        // @ts-ignore
        assert.equal(browserA.getTitle(), 'Title: WebdriverJS Testpage')

    })
```

Reply:
I think it happens because protocol commands are wrapped twice in Multiremote mode for some reason

Reply:
Managed to reproduce it locally with
```
    it('mr', () => {
        assert.equal(
            JSON.stringify(browser.getTitle()),
            JSON.stringify(['WebdriverJS Testpage', 'WebdriverJS Testpage']))
        // @ts-ignore
        assert.equal(browser.browserB.getTitle(), 'WebdriverJS Testpage')
        // @ts-ignore
        assert.equal(browser.browserA.getTitle(), 'WebdriverJS Testpage')
    })
```

Reply:
@christian-bromann I've figure out what is going on. We have to set `_NOT_FIBER` for every instance

Reply:
So it is required to revert the change and add something like this to `inFiber` function:
```
    if (Array.isArray(context.instances)) {
        context.instances.forEach(instance => {
            context[instance]._NOT_FIBER = false
        })
    }
```